File: Easy Challenges/0027 - Centuries and Leap Years/Solution.py
def Century(year):
	return (year / 100) if year % 100 == 0 else ((year - (year % 100)) / 100) + 1

def Leap_Year(year):
	return year % 400 == 0 or year % 4 == 0 and year % 100 != 0

def main():
	year = int(raw_input('Enter a year: '))
	print('Century: %d' % (Century(year)))
	print('Leap Year: %s' % ('Yes' if Leap_Year(year) else 'No'))

if __name__ == '__main__':
	main()

File: Easy Challenges/0027 - Centuries and Leap Years/Solution.java
import java.util.Scanner;

public class Solution {

    public static void main(String[] args) {
        Scanner reader = new Scanner(System.in);
        System.out.print("Enter a year: ");
        int sYear = Integer.parseInt(reader.nextLine());
        System.out.printf("Century: %d%n", century(String.valueOf(sYear)));
        System.out.printf("Leap Year: %s", leapYear(sYear) ? "Yes" : "No");
    }

    public static int century(String year) {
        return (Integer.parseInt(year) % 100 != 0) ? Integer.parseInt(year.substring(0, 2)) + 1 : Integer.parseInt(year.substring(0, 2));
    }

    public static boolean leapYear(int year) {
        return (year % 4 == 0 && year % 100 != 0) || (year % 4 == 0 && year % 100 == 0 & year % 400 == 0);
    }
    
}
